boards/arm/nrf5340dk_nrf5340/doc/index.rst

```diff
@@ -308,7 +309,7 @@ References
 
 .. _IDAU:
    https://developer.arm.com/docs/100690/latest/attribution-units-sau-and-idau
-.. _nRF5340 PDK website:
+.. _nRF5340 DK website:
```

Comment:
@ru-fu i wonder if I should just remove this link completely and re-add it once the DK web page is up..

Reply:
I would keep it, because it will most likely redirect to the DK website.